src/main/java/eu/darkbot/api/items/ItemUseResult.java

```diff
@@ -0,0 +1,65 @@
+package eu.darkbot.api.items;
+
+import eu.darkbot.api.managers.HeroItemsAPI;
+
+import java.util.function.Consumer;
+
+/**
+ * {@link HeroItemsAPI#useItem(SelectableItem, ItemUseFlag...)} results
+ */
+public enum ItemUseResult {
+    /**
+     * Item was used successfully
+     */
+    SUCCESS,
+    /**
+     * Failed to use an item
+     */
+    FAILED,
+    /**
+     * Item isn't ready
```

Comment:
But what does ready mean? defining a word with itself isn't too helpful if anyone is confused about what this means when reading our javadocs